docs/book/getting-started/deploying-zenml/manage-the-deployed-services/custom-secret-stores.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Learning how to develop a custom secret store.
+---
+
+# Custom secret stores
+
```

Comment:
can we move this to the same level as the secret-management.md file (that is, directly under deploying-zenml)? this section (`manage-the-deployed-services`) shouldn't exist anymore. in the `toc.md`, it is already positioned so.

Reply:
you would then also have to update the path in the `toc.md` file to the parent directory. Thank you for fixing my oversight! :)

Reply:
Done :+1: